python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -6780,7 +6780,12 @@ tables:
   - name: isDipole
     "@id": "#DiaSource.isDipole"
     datatype: boolean
-    description: Flag indicating diaSource is classified as a dipole
+    description: Classification of diaSource as a dipole.
+    fits:tunit:
+  - name: dipoleFitAttempted
+    "@id": "#DiaSource.dipoleFitAttempted"
+    datatype: boolean
+    description: Attempted classification of object to be a dipole.
```

Comment:
Please remove the change to this file: the dp02 schema was set when it was released, and shouldn't change after.